src/alcotest/cli.mli

```diff
@@ -25,6 +25,7 @@
 
 module type S = sig
   include Core.S
+  (** @inline *)
```

Comment:
What does `@inline` do? Is this an odoc attribute?

Reply:
Yes, it's an Odoc attribute. It has the effect of concealing the `include Core.S` banner in the documentation (which seems good, since its really an implementation detail).

Without `@inline`:

![image](https://user-images.githubusercontent.com/26125912/76603940-23099700-650e-11ea-8522-ca1b9a611cdf.png)

With `@inline`:

![image](https://user-images.githubusercontent.com/26125912/76603985-33217680-650e-11ea-837e-1a41116fe1fe.png)

Note that we can't do the same for the `Cli.S` include, since Odoc won't inline the type constraint `return = unit`.

I haven't found documentation for `@inline` anywhere; I just discovered it from usage in Jane Street packages to avoid adding documentation noise with a `_intf.ml` file (see https://github.com/janestreet/accessor/blob/master/src/subtyping.mli#L2). We could do this in a few places in the Irmin docs too...

Reply:
That's really great! We need this for Irmin docs :-)